lib/features/home/presentation/cubit/cubit/home_cubit.dart

```diff
@@ -0,0 +1,33 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../../../../launches/presentation/pages/launches_screen.dart';
+import '../../../../rockets/presentation/pages/rockets_screen.dart';
+import '../../../../upcoming/presentation/pages/upcoming_screen.dart';
+
+part 'home_state.dart';
+
+class HomeCubit extends Cubit<HomeState> {
```

Comment:
why you have two folders named cubit ?
and why it's inside presentation folder ?